deploy/components/vllm-sim-pd/kustomization.yaml

```diff
@@ -10,9 +10,3 @@ kind: Kustomization
 
 resources:
 - deployments.yaml
-
-images:
-- name: ghcr.io/llm-d/llm-d-inference-sim
-  newTag: ${VLLM_SIMULATOR_TAG}
-- name: ghcr.io/llm-d/llm-d-routing-sidecar
-  newTag: ${ROUTING_SIDECAR_TAG}
```

Comment:
Please undo this change. The file as it was allows the YAML file to be used outside of the kind based tests.